assets/src/application/ui/display/withTooltip/withTooltip.tsx

```diff
@@ -35,40 +31,39 @@ const withTooltip = <P extends withTooltipProps>(WrappedComponent: React.Compone
 				'ee-mobile-help-text--short': tooltip.length < 25,
 				'ee-mobile-help-text--long': tooltip.length > 50,
 			});
-			tooltipProps = { ...props.tooltipProps, overlayClassName: 'ee-mobile-help-text__tooltip' };
+			tooltipProps = { ...tooltipProps, className: 'ee-mobile-help-text__tooltip' };
 			toolTipped = (
 				<div className='ee-mobile-help-text__btn-wrap'>
 					<WrappedComponent
-						{...(props as P)}
-						aria-label={ariaLabel}
```

Comment:
I've added `aria-label` here because some of the chakra's components had it required.
I suppose that now it should be passed explicitly instead of being derived from `tooltip`?

Reply:
ya that's correct, the tooltip should be thought of more as "help text" which is not the same thing as an aria-label anyways
a little more work for us, but end result should be better a11y